content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -661,6 +661,76 @@ of single-resource API requests, then aggregates the responses if needed.
 By contrast, the Kubernetes API verbs **list** and **watch** allow getting multiple
 resources, and **deletecollection** allows deleting multiple resources.
 
+## Field validation
+
+  {{< feature-state for_k8s_version="v1.25" state="beta" >}}
```

Comment:
On reflection, I think this should be for the heading “Setting the field validation level” and not for the 2nd-level field validation heading. Even with the feature gate off, there's some (actually quite a lot) of field validation going on.

Reply:
moved